IframeMediaPause/iframe_media_pausing.md

```diff
@@ -145,7 +147,11 @@ window.speechSynthesis.speak(utterance);
 
 ### Interoperability with autoplay
 
-If an iframe with the permission policy property set to `allow="media-playback-while-not-rendered 'none'; autoplay *"` stops being rendered, the `media-playback-while-not-rendered` permission policy should take precedence and no media should be played while it remains not rendered. The same behavior should happen if the [`autoplay`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio#autoplay) property has been set in a HTMLMediaElement 
+This proposal does not affect autoplay behavior unless the element is not rendered, in which case all autoplay behavior should be blocked and all media playback should be paused. 
+
+For example, if an iframe with the permission policy property set to `allow="media-playback-while-not-rendered 'none'; autoplay *"` stops being rendered, the `media-playback-while-not-rendered` permission policy should take precedence and no media should be played while it remains not rendered. At the same time, if the iframe is rendered and already has `autoplay` permission either through the `autoplay` permission policy or the user agent's own criteria, then this permission state shouldn't reset when the iframe becomes not rendered. 
```

Comment:
What should be the intended behavior for frames that have the autoplay permission and then go from not rendered to rendered?

Reply:
The frame will still have autoplay permission granted and will be able again to play media without an user gesture. As long as the frame is rendered. Isn't this clear? How can I make it clearer?

Reply:
Can you just state explicitly that if the frame has autoplay permission granted and it becomes rendered, media will play automatically?

Reply:
I just want to make sure we are on the same page. The goal of this section is just to inform that "This proposal does not affect autoplay behavior unless the element is not rendered, in which case all autoplay behavior should be blocked and all media playback should be paused". The user agent is not responsible for resuming playback.

It is not part of this proposal that, if the iframe has `autoplay` permission granted and it becomes rendered, media playback will resume automatically. We are proposing that, when the iframe becomes rendered, all media will be in a paused state. Then, since the iframe has `autoplay` permission granted, it should be able to resume media playback without requiring a new user gesture.

Reply:
Do you think that your comment still apply?